contrib/completion/bash/docker

```diff
@@ -3459,6 +3461,8 @@ _docker_service_update_and_create() {
 			--dns-search-rm
 			--group-add
 			--group-rm
+			--generic-resource-add
```

Comment:
Do you want to keep this alphabetical? If so, it should be above the `--group-...` lines 🤗

Reply:
The alphabet is really comlicated...
Thanks for catching. PR is updated.